lib/mayaUsd/render/vp2RenderDelegate/meshViewportCompute.cpp

```diff
@@ -277,6 +277,12 @@ void MeshViewportCompute::createConsolidatedTopology(TopologyAccessor getTopolog
 
             HdMeshTopology& sourceTopology = getTopology(sourceMeshSharedData);
 
+            if (sourceIndex == 0) {
```

Comment:
This is to fix the assertions that follow from here. In some scenes I get different subdivision schemes than Catmull Clarke, so instead of hard coding the scheme make the scheme of the consolidated topology the same as the scheme of the first consolidated source, and then verify that all the other items have the same scheme.